GetGraphTile should return nullptr if graphid is invalid

```c++
const GraphTile* GraphReader::GetGraphTile(const GraphId& graphid);
```

invalid graphid will trigger:

```
2017/01/22 17:36:58.100516 [ERROR] src/prime_server.cpp:430 worker_t: Could not compute FileSuffix for non-existent level
```

--- a/src/baldr/graphreader.cc
+++ b/src/baldr/graphreader.cc
@@ -97,6 +97,11 @@ bool GraphReader::DoesTileExist(const boost::property_tree::ptree& pt, const Gra
 const GraphTile* GraphReader::GetGraphTile(const GraphId& graphid) {
   //TODO: clear the cache automatically once we become overcommitted by a certain amount
 
+  // Return nullptr if not a valid tile
+  if (!graphid.Is_Valid()) {
+    return nullptr;
+  }
+
   // Check if the level/tileid combination is in the cache
   auto base = graphid.Tile_Base();
   auto cached = cache_.find(base);

